src/modules/Dropdown/Dropdown.js

```diff
@@ -693,29 +711,15 @@ export default class Dropdown extends Component {
 
   open = () => {
     debug('open()')
-    debug(`dropdown is ${this.state.open ? 'already' : 'not yet'} open`)
     const { search } = this.props
-    if (search) this.refs.search.focus()
+    if (search) this._search.focus()
 
-    this.trySetState({
-      open: true,
-    }, {
-      dropdownClasses: 'active visible',
-      menuClasses: 'visible',
-    })
```

Comment:
We should only be setting the visibility classes if the open state was successful.  I moved these to their respective render functions.